src/containers/PatientDetails/DocumentPrint/styles.ts

```diff
@@ -0,0 +1,42 @@
+import { Typography } from 'antd';
+import styled from 'styled-components';
+
+const { Text, Paragraph } = Typography;
+
+export const S = {
+    HeaderSpace: styled.div`
+        @media print {
+            height: 40px;
+        }
+        height: 0;
+    `,
+    FooterSpace: styled.div`
+        @media print {
+            height: 40px;
+        }
+        height: 0;
+    `,
+    Container: styled.div`
+        @page {
+            margin: 15mm 15mm 15mm 25mm;
+        }
+        width: 100%;
+        max-width: 767px;
+        margin: 40px auto;
+        @media print {
+            max-width: 100%;
+            margin: 0 0 0 0;
+        }
```

Comment:
In css you should always put selector level attributes first and then others. We don't have linter for this case but better to remember this rule

```css
Container: styled.div`
        width: 100%;
        max-width: 767px;
        margin: 40px auto;
        @page {
            margin: 15mm 15mm 15mm 25mm;
        }
        @media print {
            max-width: 100%;
            margin: 0 0 0 0;
        }
`,
```